oauth-common/src/main/java/io/strimzi/kafka/oauth/common/HttpException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017-2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.oauth.common;
+
+import java.net.URI;
+
+public class HttpException extends RuntimeException {
+
+    private URI uri;
+    private int status;
+    private String response;
+
+    public HttpException(URI uri, int status, String response) {
```

Comment:
Do you need to include the request method too?

Reply:
Indeed.